src/main/java/openfasttrack/core/Linker.java

```diff
@@ -112,19 +111,29 @@ private void linkIgnoringRevision(final LinkedSpecificationItem item,
     {
         final List<LinkedSpecificationItem> coveredLinkedItems = this.index
                 .getByIdIgnoringVersion(id);
-        for (final LinkedSpecificationItem itemCoveredIgnoringVersion : coveredLinkedItems)
+        if (coveredLinkedItems.isEmpty())
         {
-            if (id.getRevision() < itemCoveredIgnoringVersion.getId().getRevision())
-            {
-                item.addLinkToItemWithStatus(itemCoveredIgnoringVersion, LinkStatus.OUTDATED);
-                itemCoveredIgnoringVersion.addLinkToItemWithStatus(item,
-                        LinkStatus.COVERED_OUTDATED);
-            }
-            else
+            // FIXME: Collect list of covered specification items that do not
+            // exist
+            // https://github.com/hamstercommunity/openfasttrack/issues/86
+            System.err.println("Error: " + item.getId() + " links to non-existent " + id);
+        }
+        else
+        {
+            for (final LinkedSpecificationItem itemCoveredIgnoringVersion : coveredLinkedItems)
             {
-                item.addLinkToItemWithStatus(itemCoveredIgnoringVersion, LinkStatus.PREDATED);
-                itemCoveredIgnoringVersion.addLinkToItemWithStatus(item,
-                        LinkStatus.COVERED_PREDATED);
+                if (id.getRevision() < itemCoveredIgnoringVersion.getId().getRevision())
+                {
+                    item.addLinkToItemWithStatus(itemCoveredIgnoringVersion, LinkStatus.OUTDATED);
+                    itemCoveredIgnoringVersion.addLinkToItemWithStatus(item,
+                            LinkStatus.COVERED_OUTDATED);
+                }
+                else
+                {
+                    item.addLinkToItemWithStatus(itemCoveredIgnoringVersion, LinkStatus.PREDATED);
+                    itemCoveredIgnoringVersion.addLinkToItemWithStatus(item,
+                            LinkStatus.COVERED_PREDATED);
+                }
```

Comment:
Yes and no. This code can not be reached if the revision matches.

That being said, I is sloppy of me to not create a exception in case we hit that condition.

Fixed.